.circleci/template.yml

```diff
@@ -10,6 +10,17 @@ parameters:
     default: false
 
 commands:
+  halt_if_short_flow:
+    steps:
+      - run:
+          name: Check Commit Message
+          command: |
+            COMMIT_MESSAGE=$(git log -n 1 --pretty=format:%s ${CIRCLE_SHA1})
+            if [[ "$COMMIT_MESSAGE" == *"[short-flow]"* ]]; then
```

Comment:
WDYT about supporting only a suffix `[short flow]`? Example commit:

```
Abc. [short flow]
```

Similar to [`[skip ci]`](https://circleci.com/docs/skip-build/#skip-jobs).